test/support/token_fixture.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Stripe.Fixture.Token do
+
+  def valid_card() do
+    %{card: [
+      number: "4242424242424242",
+      exp_month: exp_month(),
+      exp_year: exp_year(),
+      cvc: "314"
+    ]}
+  end
+
+  def invalid_card() do
+    %{card: [
+      number: "invalid card number",
+      exp_month: exp_month(),
+      exp_year: exp_year(),
+      cvc: "314"
+    ]}
+  end
+
+  defp exp_month() do
```

Comment:
Do we reallly need to put this in a function instead of hardcoding into valid/invalid card?